deep_water_level/src/deep_water_level/deeplabv3/utils.py

```diff
@@ -0,0 +1,78 @@
+import json
+import os
+
+import torch
+from PIL import Image
+from torch.utils.data import Dataset
+from torchvision.transforms import functional as F
+
+
+def preprocess_image(image):
+    transform = F.to_tensor(image)
+    transform = F.normalize(transform, mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
```

Comment:
did you look at the latest versions of what we're doing to preprocess in the existing codebase?
I think equalization worked much better than normalization. And also for normalization, we need to adapt it to the means and stddev of our specific dataset.

Reply:
yeah, good point, the values I used are for the pretrained models, but if we're fine-tunning I guess other values could be better